kommunicate/src/main/java/io/kommunicate/models/KmAutoSuggestionModel.java

```diff
@@ -8,8 +8,8 @@ public class KmAutoSuggestionModel extends JsonMarker {
     private String category;
     private String content;
     private boolean deleted;
-    private long created_at;
-    private long updated_at;
+    private String created_at;
```

Comment:
these are timestamp in UTC, need to find a way to store it as Date field so that anyplace if we have to use the date time, we can easily do that by converting to user's timezone

Reply:
same logic for iOS too

Reply:
okay